certified-connectors/Nextcom/apiDefinition.swagger.json

```diff
@@ -0,0 +1,585 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Nextcom",
+    "description": "Connect your application data flow to Nextcom to optimize sales and customer management.",
+    "version": "1.0",
+	"contact": {
+      "name": "Nextcom Support",
+      "url": "https://nextcom.no/en/services/support/",
+      "email": "post@nextcom.no"
+    }
+  },
+  "host": "dev.nextcom.no",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+  "application/json"
+],
+  "produces": [],
+  "paths":{
+    "/crm-system/contacts": {
+      "post": {
+        "responses": {
+          "201": {
+            "description": "Returns the new contact ID, or an existing contact ID if a duplicate was hit",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "ContactId",
+                  "title": "ContactId"
+                },
+                "insertStatus": {
+                  "type": "string",
+                  "description": "Relays the contact ID and, wether a new contact was created or if a duplicate was found and re-used.",
+                  "title": "Insert status"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "default",
+            "schema": {}
```

Comment:
@ebarland Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid response schema definition for all the operations in the swagger.

Reply:
added response schemas for all outputs: 
https://github.com/microsoft/PowerPlatformConnectors/pull/1123/commits/2a4fad1485e9a0b2f9d5fb2494a3f37236b3d607

This did give me warnings in the CLI validation tool though: 

> Swagger certification succeeded with warnings. It is not required but recommended to fix following warnings: 
> Warning : paths/~1crm-system~1contacts/post/responses : The operation 'NX_CONTACT' has more than one response with specified schema, only one schema will be used in the designer.
> Warning : paths/~1crm-system~1pipelines~1contracts/post/responses : The operation 'NX_PIPELINE' has more than one response with specified schema, only one schema will be used in the designer.